ggml.c

```diff
@@ -28,12 +28,7 @@
 
 #if defined _MSC_VER || defined(__MINGW32__)
```

Comment:
Unrelated to this PR, but this should probably be
```suggestion
#if defined(_WIN32)
```
The condition isn't the toolchain used, but the target.